cmd/manager/main.go

```diff
@@ -202,9 +209,9 @@ func main() {
 // namespace.
 func newConfigManagedClient(mgr manager.Manager) (runtimeclient.Client, manager.Runnable, error) {
 	cacheOpts := cache.Options{
-		Scheme:     mgr.GetScheme(),
-		Mapper:     mgr.GetRESTMapper(),
-		Namespaces: []string{awsclient.KubeCloudConfigNamespace},
+		Scheme:            mgr.GetScheme(),
+		Mapper:            mgr.GetRESTMapper(),
+		DefaultNamespaces: map[string]cache.Config{},
```

Comment:
```suggestion
		DefaultNamespaces: map[string]cache.Config{
		    awsclient.KubeCloudConfigNamespace: {},
		},
```

I think you omitted the namespace in there.

Reply:
You're right! Thanks, added the namespace back